tools/firtool/firtool.cpp

```diff
@@ -660,6 +660,8 @@ static LogicalResult processBuffer(
     pm.nest<firrtl::CircuitOp>().nest<firrtl::FModuleOp>().addPass(
         firrtl::createFlattenMemoryPass());
 
+  if (!disableCheckCombCycles)
+    pm.nest<firrtl::CircuitOp>().addPass(firrtl::createCheckCombLoopsPass());
```

Comment:
Don't we need to run after LowerTypes and ExpandWhrens? If so, I think there is no bulk connection; LowerTypes will expand connections into ground types.

Reply:
I think, bulk connect is the only limitation because of which we have to move it after ExpandWhens. I will handle the bulk connects next week and then try to move this pass before LowerTypes.

Reply:
For now, I will move it after ExpandWhens, so that this pass can be enabled by default.